cdap-api/src/main/java/co/cask/cdap/api/workflow/Value.java

```diff
@@ -53,6 +56,13 @@ public long getAsLong() {
   }
 
   /**
+   * @return the double value
+   */
+  public double getAsDouble() {
```

Comment:
So this method will throw `NumberFormatException` if the value is not a double. It's better document it. Or you can simply say, it returns the result of calling `Double.parseDouble(toString())`. Document similarly for other conversion methods.